tests/testsuite/features_namespaced.rs

```diff
@@ -920,18 +944,17 @@ fn publish_no_implicit() {
 
     p.cargo("publish --no-verify")
         .replace_crates_io(registry.index_url())
-        .with_stderr(
-            "\
-[UPDATING] [..]
-[PACKAGING] foo v0.1.0 [..]
-[PACKAGED] [..]
-[UPLOADING] foo v0.1.0 [..]
-[UPLOADED] foo v0.1.0 [..]
-[NOTE] waiting [..]
-You may press ctrl-c [..]
-[PUBLISHED] foo v0.1.0 [..]
-",
-        )
+        .with_stderr_data(str![[r#"
+[UPDATING] crates.io index
+[PACKAGING] foo v0.1.0 ([ROOT]/foo)
+[PACKAGED] 3 files, [FILE_SIZE]B ([FILE_SIZE]B compressed)
```

Comment:
We might consider redacting the size of files.
FYI: https://github.com/rust-lang/cargo/pull/14096#discussion_r1644832897

Reply:
Since we've decided not to auto-redact the number of files, I'll mark this as resolved now. Feel free to reopen it if needed!